frontend/lib/src/components/widgets/DataFrame/hooks/useDataLoader.ts

```diff
@@ -86,7 +88,10 @@ function useDataLoader(
 
       try {
         // Arrow has the header in first row
```

Comment:
nit: perhaps extend this comment to `// Arrow has the header in first row which is why we skip all header rows to get to the data`

Reply:
Changed